python/lsst/afw/geom/angle.cc

```diff
@@ -0,0 +1,125 @@
+    mod.def("AngleUnit_mul", [](lsst::afw::geom::AngleUnit const& lhs, double rhs) -> Angle {
+        return rhs*lhs;
+    });
+    mod.def("AngleUnit_mul", [](double lhs, lsst::afw::geom::AngleUnit const& rhs) -> Angle {
+        return lhs*rhs;
+    });
+
+    /* Members */
+    mod.attr("radians") = py::cast(new AngleUnit(1.0));
+    mod.attr("degrees") = py::cast(new AngleUnit(PI/180.0));
+    mod.attr("hours") = py::cast(new AngleUnit(PI*15.0/180.0));
+    mod.attr("arcminutes") = py::cast(new AngleUnit(PI/60/180.0));
+    mod.attr("arcseconds") = py::cast(new AngleUnit(PI/180.0/3600.0));
+
+    py::class_<Angle> clsAngle(mod, "Angle");
+    
+    /* Constructors */
+    clsAngle.def(py::init<double, AngleUnit>(), py::arg("val"), py::arg("units")=radians);
+    clsAngle.def(py::init<>());
+
+    /* Operators */
+    clsAngle.def(py::self *= float());
+    clsAngle.def(py::self *= int());
+    clsAngle.def(py::self += float());
+    clsAngle.def(py::self += int());
+    clsAngle.def(py::self -= float());
+    clsAngle.def(py::self -= int());
+    clsAngle.def(py::self == py::self);
+    clsAngle.def(py::self != py::self);
+    clsAngle.def(py::self <= py::self);
+    clsAngle.def(py::self >= py::self);
+    clsAngle.def(py::self < py::self);
+    clsAngle.def(py::self > py::self);
+    clsAngle.def(py::self - py::self);
+    clsAngle.def(py::self + py::self);
+    clsAngle.def("__float__", &Angle::operator double);
+    clsAngle.def("__div__", [](Angle &a, double d) {
+        return a / d;
+    });
+    clsAngle.def("__rdiv__", [](Angle &a, double d) {
+        return a / d;
+    });
+    mod.def("Angle_mul", [](Angle &lhs, Angle &rhs) {
+        return lhs * rhs;
+    });
+    mod.def("Angle_mul", [](Angle &lhs, double rhs) {
+        return lhs * rhs;
+    });
+    mod.def("Angle_mul", [](Angle &lhs, int rhs) {
+        return lhs * rhs;
+    });
+
+    /* Members */
+    clsAngle.def("asAngularUnits", &Angle::asAngularUnits);
+    clsAngle.def("asRadians", &Angle::asRadians);
+    clsAngle.def("asDegrees", &Angle::asDegrees);
+    clsAngle.def("asHours", &Angle::asHours);
+    clsAngle.def("asArcminutes", &Angle::asArcminutes);
+    clsAngle.def("asArcseconds", &Angle::asArcseconds);
+    clsAngle.def("toUnitSphereDistanceSquared", &Angle::toUnitSphereDistanceSquared);
+    clsAngle.def("fromUnitSphereDistanceSquared", &Angle::fromUnitSphereDistanceSquared);
+    clsAngle.def("wrap", &Angle::wrap);
+    clsAngle.def("wrapCtr", &Angle::wrapCtr);
+    clsAngle.def("wrapNear", &Angle::wrapNear);
+
+    /* Non members */
+    mod.attr("PI") = py::float_(PI);
+    mod.attr("TWOPI") = py::float_(TWOPI);
+    mod.attr("HALFPI") = py::float_(HALFPI);
+    mod.attr("ONE_OVER_PI") = py::float_(ONE_OVER_PI);
+    mod.attr("SQRTPI") = py::float_(SQRTPI);
+    mod.attr("INVSQRTPI") = py::float_(INVSQRTPI);
+    mod.attr("ROOT2") = py::float_(ROOT2);
+    mod.attr("NullAngle") = py::cast(new Angle(-1000000., degrees));
```

Comment:
Why can't this use the definition from the `Angle.h` header file? It seems dangerous to define it in C++ and in Python.